examples/node/transfer-ae.mjs

```diff
@@ -80,3 +80,10 @@ console.log(`Balance of ${recipient} (after): ${balanceAfter} aettos`);
 //  - If the recipient is the same account as the sender (default of the script if no arguments
 //    provided) the balance will be lower after transfer because a transaction `fee` has been
 //    paid to the miners.
+
+// ## 7. Transfer a fraction of your AE to another account
+// Instead of `spend` function you can use the `transferFunds` function
+// to transfer a fraction of your AE to another account
+const fraction = 1;
```

Comment:
maybe provide an additional hint that "1" will transfer 100% and "0.5" would transfer 50% to eliminate any confusion :)